azure-application-insights-spring-boot-starter/build.gradle

```diff
@@ -47,14 +47,16 @@ sourceSets {
 
 def springBootVersion = '1.5.21.RELEASE'
 dependencies {
-    compile(project(':core'))
-    compile(project(':web'))
+    compile(project(path: ':core', configuration: 'shadow'))
```

Comment:
Do we need `:core` since it's included in `:web` shadow configuration?

Reply:
Maybe not. I'll give it a shot and if the tests fail, i'll revert.

Reply:
Looks like it worked